test/package-test.js

```diff
@@ -0,0 +1,172 @@
+'use strict';
+
+const Hapi = require('hapi');
+const lab = exports.lab = require('lab').script();
+const code = exports.code = require('code');
+const hapiSlack = require('../');
+let server;
+
+lab.afterEach((done) => {
+  server.stop(() => {
+    done();
+  });
+});
+lab.test('converts a basic message passed as string ', (done) => {
+  server = new Hapi.Server({});
+  server.connection({ port: 8080 });
```

Comment:
is this required? would be great to have potential port conflicts when spinning up tests